calva/repl-window.ts

```diff
@@ -26,14 +26,9 @@ export function isReplWindowVisible(mode: "clj" | "cljs" = "clj") {
     if (!replWindows[mode]) {
         return(false);   
     }
-    try {
-        // the webview may already be disposed.
-        if(!replWindows[mode].panel.visible) {
-            return(false);  
-        }
-    } catch(e) {
-        return(false);
-    }
+    // Do not check for visability because the REPL window can 
+    // exist and be obscured in the editor bar. See if it exists 
+    // but not if it is visible.
```

Comment:
This function needs another name after the change. :smile:

Reply:
@PEZ `existsReplWindow() `?